utils/src/main/scala/za/co/absa/enceladus/utils/explode/ExplodeTools.scala

```diff
@@ -213,9 +214,9 @@ object ExplodeTools {
       // Drop the array size column
       .drop(col(explosion.sizeFieldName))
       // restore original record order
-      .orderBy(groupedCol)
+      .orderBy(orderByRecordCol)
       // remove monotonic id created during explode
-      .drop(groupedCol)
+      .drop(orderByRecordCol)
   }
   // scalastyle:on method.length
```

Comment:
is this IDE configuration?

Reply:
Not just IDE, scalastyle produces a report with warnings and errors found, unless they are wrapped in comments like these. For legitimate cases (a method can only be as simple as its use case) where exceptions need to be made, this prevents unnecessary warnings from stacking up.